views/layout.erb

```diff
@@ -68,7 +126,26 @@
   </div>
 </footer>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-    <!-- Include all compiled plugins (below), or include individual files as needed -->
-    <link rel="stylesheet" type="text/css" href="<%= url("/bootstrap-3.3.7/css/bootstrap.min.css") %>">
+    <script type="text/javascript">
+        var refreshImages, startRefresh;
+
+        startRefresh = function(node) {
+          return node.src = URL(node.src).setQuery("timestamp", (new Date).getTime()).toString();
+        };
+        // anonymous function to update all visible Ganglia graphs every 5 seconds
+        (refreshImages = function() {
+          var i, nodes;
+          nodes = $('.updateable');
+          i = 0;
+          while (i < nodes.length) {
+            if ($(nodes[i]).is(':visible')) {
+              startRefresh(nodes[i]);
+            }
+            i++;
+          }
+          setTimeout(refreshImages, 5000);
+        })();
```

Comment:
I know I said to move this to the layout, but this looks like it is specific to the ganglia graphs, so maybe this block should just be at the bottom of the ganglia graphs view.

Also, I think this block can be refactored and simplified. The "refreshImages" function is essentially

1. "get all of the updatable elements that are visible", and 
2. call "startRefresh" with these as an argument

The fix:

1. For "get all of the updatable elements that are visible", `$('.updateable:visible')` should work. No need for a while loop.
2. The array [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) passes the value as the first argument, so this might work too: `$('.updateable:visible').toArray().forEach(startRefresh)`

At this point the code might be easier to read if the body of `startRefresh` be in an anonymous function instead of a function expression (setting function to variable). Then you could use jQuery's built in each method (I believe `this` is the value in that case) i.e. `$('.updateable:visible').each(function(){ ... });`